RELEASE.md

```diff
@@ -214,6 +207,13 @@ Run Dangerzone against a list of documents, and tick all options. Ensure that:
 Run Dangerzone against a set of documents that you drag-n-drop. Files should be
 added and conversion should run without issue.
 
+When testing on Linux with the `dev_scripts/env.py` script, you will need to install `thunar` and run it:
+
+```bash
+./dev_scripts/env.py --distro ubuntu --version 24.04 run bash -C "aptitude install thunar"
+
+```
```

Comment:
Actually, I plan to install `thunar` in our dev environment, it's simpler that way.